physx/src/geometry.rs

```diff
@@ -13,8 +13,7 @@
 
 use physx_sys::*;
 
-use glam::Vec3;
-pub type Point3 = Vec3;
```

Comment:
@Chris-Nicholls I'm hesitant about this change, since it means we're leaking a Px type in a single API. Maybe this should still use the Glam type, but the Cook handles running with SSE glam. As far as I can tell, we should be able to just change the stride in the points data passed into the cooker if we're using SSE, unless the member ordering changes, in which case we'd have to transform the input into a new vector. I don't think that's too bad since you shouldn't cook trimeshes in the hot path, but...

@repi or @h3r2tic Do you want to weigh in?

Reply:
Maybe just fall back to [f32; 3] as the trimesh vertex type?   They can be trivially converted to and from glam Vec3 using into() anyway, so aren't too inconvenient